velox/vector/fuzzer/VectorFuzzer.cpp

```diff
@@ -431,7 +431,7 @@ VectorPtr VectorFuzzer::fuzzConstant(const TypePtr& type, vector_size_t size) {
   // generated inner vector.
 
   // Inner vector size can't be zero.
-  auto innerVectorSize = rand<uint32_t>(rng_) % opts_.vectorSize + 1;
```

Comment:
@pramodsatya : It would be better to use something like (opts_.vectorSize + 1) to ensure we avoid mod 0, but still retain some randomness. But that assumes that vectorSize cannot be -ve. We should validate that wherever vectorSize is initialized.